rpcs3/rpcs3qt/memory_viewer_panel.cpp

```diff
@@ -179,15 +183,11 @@ memory_viewer_panel::memory_viewer_panel(QWidget* parent, u32 addr)
 	hbox_mem_panel->addWidget(m_mem_ascii);
 	hbox_mem_panel->addSpacing(10);
 
-	//Memory Panel: Set size of the QTextEdits
-	m_mem_hex->setFixedSize(QSize(pSize * 3 * m_colcount + 6, 228));
-	m_mem_ascii->setFixedSize(QSize(pSize * m_colcount + 6, 228));
```

Comment:
I found that those were redundent because resizing is all handled in ShowAddress().